src/views/AgmLocationChg.vue

```diff
@@ -419,7 +420,7 @@ export default class AgmLocationChg extends Mixins(CommonMixin, DateMixin,
   mounted (): void {
     // Pre-populate the certified block with the logged in user's name (if not staff)
     if (!this.isRoleStaff && this.getUserInfo) {
-      this.certifiedBy = this.getUserInfo.firstname + ' ' + this.getUserInfo.lastName
+      this.certifiedBy = this.getUserInfo.firstname + ' ' + this.getUserInfo.lastname
```

Comment:
So sorry about this 😊 Thanks for fixing it.

Reply:
No worries Sev! Happened to me too when I first wrote the variables. I had `firstName` and `lastName` so I was getting undefined undefined 😆